### PR TITLE
Imp supint 1759 new typography

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

File: ui/nuxeo-user-group-management/nuxeo-create-group.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ import '../nuxeo-button-styles.js';
 
         <div class="header">
           <iron-icon icon="nuxeo:group"></iron-icon>
-          <h3>[[i18n('createGroup.heading')]]</h3>
+          <h3 class="headline-4">[[i18n('createGroup.heading')]]</h3>
```
Comment (Gabez0r) [inline ui/nuxeo-user-group-management/nuxeo-create-group.js:96]: No need to applying the `headline-4` class here, as we could just switch this `h3` to an `h4`.

File: ui/nuxeo-user-group-management/nuxeo-group-management.js [hunk 2]
```diff
@@ -99,6 +99,13 @@ import '../nuxeo-button-styles.js';
             margin-right: 0.2rem;
           }
 
+          h3 {
```
Comment (Gabez0r) [inline ui/nuxeo-user-group-management/nuxeo-group-management.js:102]: Isn't this already coming from `nuxeo-styles`?

File: ui/widgets/nuxeo-card.js [hunk 3]
```diff
@@ -93,10 +93,14 @@ import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
...
           .header .heading {
             @apply --layout-flex;
-            font-size: 1rem;
-            font-weight: 700;
-            letter-spacing: 0.04em;
-            text-transform: uppercase;
+            font-weight: 400;
+            font-size: 1.25rem;
+            letter-spacing: 0;
+            line-height: 1.625rem;
```
Comment (Gabez0r) [inline ui/widgets/nuxeo-card.js:99]: Isn't this the style applied to the `h3`? Maybe turning `.header` into a span and `.heading` into an `h3` would help reuse the style?

File: ui/widgets/nuxeo-card.js [hunk 4]
```diff
@@ -107,9 +111,9 @@ import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
 
         <dom-if if="[[_hasHeading(icon, heading, collapsible)]]">
           <template>
-            <h3 class="header" on-click="_toggle">
+            <h3 class="headline-4" on-click="_toggle">
```
Comment (Gabez0r) [inline ui/widgets/nuxeo-card.js:114]: We could just change this to an `h4`.

File: core/demo/nuxeo-page-provider.html [hunk 5]
```diff
@@ -21,7 +21,7 @@
     <script src="/components/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
   </head>
 
-  <style>
+  <style include="nuxeo-styles">
```
Comment (Gabez0r) [inline core/demo/nuxeo-page-provider.html:24]: Where would this come from? Are we importing it anywhere?

File: ui/actions/nuxeo-add-to-collection-button.js [hunk 6]
```diff
@@ -50,7 +50,7 @@ import '../nuxeo-button-styles.js';
   class AddToCollectionButton extends mixinBehaviors([I18nBehavior, FiltersBehavior], Nuxeo.Element) {
     static get template() {
       return html`
-        <style include="nuxeo-action-button-styles nuxeo-button-styles">
+        <style include="nuxeo-action-button-styles nuxeo-button-styles nuxeo-styles">
```
Comment (Gabez0r) [inline ui/actions/nuxeo-add-to-collection-button.js:53]: What styles do we need from nuxeo-styles, here?

**Review by Gabez0r — COMMENTED** [review]
From the point we are including `nuxeo-styles` everywhere, this means that nuxeo elements will be tied to a style module that does not exist on the library. If this is really needed, then we should consider moving it to this repo.

**Comment by semisse** [comment]
Closing this PR since the actual work is being done under https://github.com/nuxeo/nuxeo-elements/pull/305

**Pull request closed without merging**